portfolio/src/main/webapp/js/comments.js

```diff
@@ -0,0 +1,51 @@
+
+/**
+ * Handle displaying (and maintaining state for) comments on individual projects.
+ */
+export class Comment {
```

Comment:
In hindsight I totally agree. There are now a ton of chained PRs that depend on this API, do you thing it would be easier to change it here (and patch all chained PRs) or refactor this if and when everything else is merged?

Reply:
Spoke offline about this, decided to patch at the end.